server/src/main/java/com/exacaster/lighter/application/Application.java

```diff
@@ -65,6 +68,14 @@ public SubmitParams getSubmitParams() {
         return submitParams;
     }
 
+    public boolean isDeleted() {
+        return deleted;
+    }
+
+    public boolean isNotDeleted() {
```

Comment:
I think `@JsonIgnore` should be added on this one. We do not want to have both "isDeleted" & "isNotDeleted"  on  the json representation of application, when we return it on API endpoints. 

It can be added on `isDeleted()` as well, I think there are no cases when we return deleted applications on the http api layer.

Reply:
right

Reply:
done